autopush/router/apnsrouter.py

```diff
@@ -44,7 +44,7 @@ def __init__(self, ap_settings, router_conf):
         self._connect()
         self.log.debug("Starting APNS router...")
 
-    def register(self, uaid, router_data, *args, **kwargs):
+    def register(self, uaid, router_data={}, *args, **kwargs):
```

Comment:
we really don't want the mutable default (router_data={}) here. router_data seems to be always be required, so it should continue being a positional argument to enforce that